### ISSUE TITLE
Participation marks query

### ISSUE THREAD

**Issue by nikkiDEEE** [nus-cs2113-AY2223S2/forum#15]
As long as we submit the Coursemology programming exercise the midnight before the tutorial, do I get the full participation marks ? The submission before monday (end of lecture + 4 days) only applies to quizzes right? Just clarifying, thank you

**Comment by okkhoy** [comment]
Yes. As per the details here: https://nus-cs2113-ay2223s2.github.io/website/admin/participation.html#participation-marks

**Comment by WilsonLee2000** [comment]
Hi professor, 
1) the quizzes will be conducted on CANVAS?
2) For the coursemology programming exercises bonus marks, aren't they counted towards participation marks as well? (for the current exercise: we need to submit early by 30 January 2359 to get that bonus mark.) But 30 January 2359 is not the midnight before my tutorial hence I'm confused.

**Comment by okkhoy** [comment]
1. Yes
2. I will update the deadline to be Friday instead (to subsume all tutorials). Bonus marks is just a way to see if you finished within the week. No bonus mark is awarded for these. **

** I think I responded to someone earlier that it is the early submission time - I stand corrected.
